templates/utils.js

```diff
@@ -1,10 +1,20 @@
+// https://github.com/GoogleChromeLabs/jsbi/issues/30#issuecomment-1006086291
+(BigInt.prototype.toJSON = function () {
+  return this.toString();
+});
+
+
 export const withDefaults =
   (template, expectedArgsDefaults, debug = false) =>
   (receivedArgs) => {
     const argsWithDefault = Object.fromEntries(
       Object.entries(expectedArgsDefaults)
-      .map(([argName, argDefault]) => [argName, receivedArgs[argName] ?? [argDefault]])
-    );
+        .map(([argName, argDefault]) => [
+          argName,
+          (receivedArgs[argName] ?? [argDefault])
+            .map(arg => typeof arg === 'string' ? arg : JSON.stringify(arg))
```

Comment:
main change: Stgringify variable if needed. Nothing breaks for now since we are always passed strings